src/error/category.js

```diff
@@ -28,7 +28,10 @@ const Category: CategoryType = {
   'ADS': 8,
 
   /** Errors in the database storage (offline). */
-  'STORAGE': 9
+  'STORAGE': 9,
+
+  /** Errors from a plugin. */
+  'PLUGIN': 11
```

Comment:
no need for plugin name-spacing, too generic to have the  `PLUGIN_FATAL_ERROR`, we already have a fatal error convention via the `SEVERITY` so it's also redundant to create an event with `FATAL` naming IMO.